### PR TITLE
fix: remove geoip-lite for ip3country"

### PR DESCRIPTION
**Pull request by busla**
# Remove [geoip-lite](https://www.npmjs.com/package/geoip-lite) package for [ip3country](https://www.npmjs.com/package/ip3country) (less overhead)

- Step 1/2 for using Cloudfront-Viewer-Country headers to not have to depend on a package for determining ip locations.
- Step2 will be added in [this PR](https://github.com/island-is/island.is/pull/14145)

## What
`geoip-lite` is extremely cumbersome to use and uses exessive compute resources when updating the geoip db when services start and requires a mounted volume.

This PR removes `geoip-lite` for `ip3country`, a way less resource heavy package.

## Why
Originally we wanted to remove the `geoip-lite`  [PR #12371](https://github.com/island-is/island.is/pull/12371), by eliminating the need for bulky geoip data handling, but that caused other problems.

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/infra-nest-server/src/lib/bootstrap.ts [hunk 1]
```diff
@@ -28,8 +28,12 @@ export const createApp = async ({
   appModule,
   enableVersioning,
   healthCheck,
+  beforeAppInit,
```
Comment (AndesKrrrrrrrrrrr) [inline libs/infra-nest-server/src/lib/bootstrap.ts:31]: Consider a better name, e.g. `appInit`, `initHook`, `preBootFunc`

File: libs/infra-nest-server/src/lib/types.ts [hunk 2]
```diff
@@ -66,6 +66,11 @@ export type RunServerOptions = {
    * Otherwise an object can be provided to override specific options.
    */
   healthCheck?: boolean | HealthCheckOptions
+
+  /**
+   * Hook to run before app is initialized.
+   */
+  beforeAppInit?: () => Promise<void>
```
Comment (AndesKrrrrrrrrrrr) [inline libs/infra-nest-server/src/lib/types.ts:73]: :point_up:

**Review by AndesKrrrrrrrrrrr — APPROVED** [review]
Minor comments, otherwise LGTM

File: libs/infra-nest-server/src/lib/types.ts (hunk 2 above)
Comment (busla) [reply libs/infra-nest-server/src/lib/types.ts:73]: thanks, but I am sticking with `beforeAppInit`

File: apps/services/sessions/docker-compose.yml [hunk 3]
```diff
@@ -7,7 +7,7 @@ services:
     networks:
       - local
     volumes:
-      - ./local-db/:/docker-entrypoint-initdb.d/
+      - ./local-db/:/docker-entrypoint-initdb.d/:Z
```
Comment (eirikurn) [inline apps/services/sessions/docker-compose.yml:10]: Typo? What does this do?

File: libs/infra-nest-server/src/lib/types.ts (hunk 2 above)
Comment (eirikurn) [reply libs/infra-nest-server/src/lib/types.ts:73]: Consider using NestJS lifecycle events:

https://docs.nestjs.com/fundamentals/lifecycle-events

**Review by eirikurn — APPROVED** [review]
LGTM with minor comments.

**Pull request merged**